docs/README.md

```diff
@@ -20,56 +20,58 @@ Thank you!
 
 ### Test credentials
 
-Please note that not all payment methods support testing. The payment methods that support testing payments are enabled.
+Please note that not all payment methods support testing, so only the payment methods that support testing payments are enabled for these credentials.
 
 * Merchant Id: `375917`
 * Secret Key: `SAIPPUAKAUPPIAS`
 
 ### HTTP Response summary
 
-General API http status codes and what to expect of them.
+General API HTTP status codes and what to expect of them.
 
-* 200 - OK - Everything worked as expected.
-* 400 - Bad Request - The request was unacceptable, probably due to missing a required parameter.
-* 401 - Unauthorized - Hmac calculation failed or Merchant has no access to this feature.
-* 404 - Not Found - The requested resource doesn't exist.
+Code | Text | Description
+---- | ---- | -----------
+200  | OK   | Everything worked as expected.
+400  | Bad Request |The request was unacceptable, probably due to missing a required parameter.
+401  | Unauthorized | HMAC calculation failed or Merchant has no access to this feature.
+404  | Not Found | The requested resource doesn't exist.
 
 ### Headers and request signing
 
-All the API calls need to be signed using HMAC and SHA256 or SHA512.
+All API calls need to be signed using HMAC and SHA256 or SHA512.
 
-HMAC is calculated from the http requests body payload using the signing secret. Header key's must be sorted alphabetically and included in hmac calculation.
+HMAC is calculated from the HTTP request's body payload using the signing secret. Header keys must be sorted alphabetically, lower-cased and included in HMAC calculation.
 
-* Merchant ID needs to be in `Checkout-Account`
-* Algorithm needs to be in `Checkout-Algorithm` (sha256 or sha512)
-* Calculated HMAC needs to be send in each request in http-header named `Checkout-Signature`
-* Method header `Checkout-Method` needs to be set to `POST, GET OR DELETE`
+* The merchant ID needs to be in `Checkout-Account`
+* The algorithm needs to be in `Checkout-Algorithm` (SHA256 or SHA512)
+* The calculated HMAC needs to be sent in each request in the HTTP header `Checkout-Signature`
+* The method header `Checkout-Method` needs to be set to `POST`, `GET` or `DELETE`
 
 So the signature is calculated from these values as in this example:
 
 ```
 checkout-account:1234\n
-checkout-algorithm:sha512\n
+checkout-algorithm:SHA512\n
 checkout-method:POST\n
 checkout-nonce:1234\n
 checkout-timestamp:2018-07-05T11:19:25.950Z\n
 REQUEST BODY
 ```
 
-These are passed to hmac function which uses SHA512 algorithm. Carriage returns (\r) should not be used, only line feed (\n).
+These are passed to the HMAC function which uses SHA512 algorithm. Carriage returns (\r) should not be used, only line feed (\n).
```

Comment:
No conditionals.

So no "should", rather say:

Do not use (\r) carriage returns, only line feed (\n).

For the implemented we are the only and single source of truth so conditionals cause confusion.

Reply:
Or taking it even further:
Only use line feed (\n).